src/domains/transaction/components/SearchRecipient/SearchRecipient.tsx

```diff
@@ -63,16 +63,15 @@ const SearchRecipientListItem: FC<SearchRecipientListItemProperties> = ({
 				<Address
 					walletName={recipient.alias}
 					address={recipient.address}
-					truncateOnTable
```

Comment:
This breaks table on ~800px breakpoint

![image](https://github.com/user-attachments/assets/2a9bf8c4-1e42-4e96-b9ef-d7c254527b35)